Utilities/drone_exceptions.py

```diff
@@ -0,0 +1,71 @@
+class NetworkException(Exception):
+    """
+    Any network or connection errors
+    """
+    pass
+
+
+class AltitudeException(Exception):
+    """
+    Too low or too high altitude
+    """
+    pass
+
+
+class ThrustException(Exception):
+    """
+    Too low or too high thrust
+    """
+    pass
+
+
+class VelocityException(Exception):
+    """
+    Too low or too high velocity
+    """
+    pass
+
+
+class BadArgumentException(Exception):
+    """
+    Caused by wrong call to function or something
+    """
+    pass
+
+class EmergencyLandException(Exception):
+    """
+    Caused by wrong call to function or something
```

Comment:
Was this intentional? It seems like this description should be akin to:

Thrown when an unrecoverable state is reached and the drone should land for preservation of its safety and the safety of others.

Reply:
It was not and I have changed all of the Exception docstrings to be a bit more descriptive and uniform.